cbrain_task/civet_combiner/bourreau/civet_combiner.rb

```diff
@@ -144,6 +144,52 @@ def cluster_commands #:nodoc:
   end
 
   def save_results #:nodoc:
+
+    if params[:copy_files]
```

Comment:
I think the `if else` should surround just the part that copy the CivetOutputs into the study object, so that all the rest of the code before and after stay the same. In the current code we're missing all the provenance information that was in the original save_results. that is,

```ruby
def save_result
  # original code here
  klass = copy ? CIvetStudy : CivetVirtualStudy
  study = klass.new
  # more original code
  if copy
     # rsync stuff in a private method
  else
     set_civet_output()
  end
  # original code with provenance here
end
```